ichnaea/conf.py

```diff
@@ -30,6 +38,14 @@ class AppConfig(RequiredConfigMixin):
         parser=bool,
         doc="Whether or not we are running tests.",
     )
+    required_config.add_option(
+        "logging_level",
+        default="INFO",
```

Comment:
I think we want INFO in stage/production. I think we want to keep this value and then go through and fix the logging so that we get what we want if stage/production are at INFO level.